libs/core/algorithms/include/hpx/parallel/algorithms/detail/generate.hpp

```diff
@@ -0,0 +1,87 @@
+//  Copyright (c) 2021 Srinivas Yadav
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config.hpp>
+#include <hpx/executors/execution_policy.hpp>
+#include <hpx/functional/tag_fallback_dispatch.hpp>
```

Comment:
We renamed it to `tag_invoke` again :upside_down_face: sorry...

Reply:
Np ;) , I will make the required changes and push!